go/client/pgp_ui.go

```diff
@@ -23,7 +23,12 @@ func NewPgpUIProtocol(g *libkb.GlobalContext) rpc.Protocol {
 	return keybase1.PGPUiProtocol(g.UI.GetPgpUI())
 }
 
-func (p PgpUI) OutputSignatureSuccess(_ context.Context, arg keybase1.OutputSignatureSuccessArg) error {
+func (p PgpUI) OutputPGPWarning(_ context.Context, arg keybase1.OutputPGPWarningArg) error {
+	_, err := p.w.Write([]byte(ColorString(p.G(), "red", fmt.Sprintf("WARNING: %s\n", arg.Warning))))
+	return err
```

Comment:
I don't think we care about failures to write to output stream in pgp_ui (see line 37: `_, _ = p.w.Write([]byte(s))`). But cc @patrickxb on that